plugins/modules/helm.py

```diff
@@ -496,7 +496,8 @@ def main():
             changed = True
 
         elif force or release_values != release_status['values'] \
-                or (chart_info['name'] + '-' + chart_info['version']) != release_status["chart"]:
+                or (chart_info['name'] + '-' + chart_info['version']) != release_status["chart"] \
+                or chart_info['appVersion'] != release_status["app_version"]:
```

Comment:
Are these values guaranteed to be present?

Reply:
More discussion in https://github.com/ansible-collections/community.kubernetes/pull/247#discussion_r500382019